utils/post.js

```diff
@@ -131,6 +131,10 @@ const modifyPollPostObject = async (userId, item) => {
   post.pollOptions = pollOptions;
   post.voteCount = voteCount;
 
+  if (post.anonimity) {
+    post.isBlurredPost = isBlurredPost;
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `if (post.anonimity)` checks if the post has anonymity enabled and then sets the `isBlurredPost` property. However, this might not be the correct logic because a post can be anonymous but not necessarily blurred. It would be better to directly assign the `isBlurredPost` value without checking for `post.anonimity`.
```diff
-  if (post.anonimity) {
-    post.isBlurredPost = isBlurredPost;
-  }
+  post.isBlurredPost = isBlurredPost;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->